chef/cookbooks/ceph/providers/pool.rb

```diff
@@ -0,0 +1,23 @@
+def whyrun_supported?
+  true
```

Comment:
You are right, I was wondering for what reason we need this in `ceph_client` LWRP too.